go/mcap/writer_test.go

```diff
@@ -142,6 +143,80 @@ func TestOutputDeterminism(t *testing.T) {
 	}
 }
 
+func TestChannelsOutsideChunks(t *testing.T) {
+	buf := &bytes.Buffer{}
+	w, err := NewWriter(buf, &WriterOptions{
+		Chunked:               true,
+		ChunkSize:             1024,
+		Compression:           CompressionNone,
+		IncludeCRC:            true,
+		ChannelsOutsideChunks: true,
+	})
+	require.NoError(t, err)
```

Comment:
what is this require business?

Reply:
I learned about this recently because my linter started complaining about usage of assert in dp3. Probably when you guys upgrade you'll get warnings too. I don't mind changing this to assert though - will do that.

As far as I understand it this causes the test to bail right away instead of plowing through, but I haven't actually verified the behavior, just that the linter was happy.